src/boot/efi/meson.build

```diff
@@ -55,6 +55,7 @@ if not cc.has_header_symbol('efi.h', 'EFI_IMAGE_MACHINE_X64',
 endif
 
 objcopy = run_command(cc.cmd_array(), '-print-prog-name=objcopy', check: true).stdout().strip()
+objcopy_version = run_command(objcopy, '--version', check: true).stdout().split('\n')[0].split(' ')[-1]
```

Comment:
`find_program(objcopy, version : '>=2.38', required:false)` does the same and is idiomatic.

Reply:
Surely, because we need to support objcopy < 2.38 as well find_program() would need to be called twice (lines 57/58) and would subsequently require a new call to find_program() every time we need to check whether we have an objcopy greater than a certain version. The version attribute (not dependency) of find_program() which would be useful was added in meson 0.62.0 but systemd targets 0.52.0.  Unless, I am missing something.

Reply:
The docs say it's 0.52.0. No need to call this twice, you just leave the first line as is and use find_program to check the version. All this stuff is planned to go away anyways, so no point in hypothetical futures.

Reply:
done